docker-compose.yml

```diff
@@ -114,6 +114,16 @@ services:
       - nexus
       - rabbitmq
 
+  jaeger:
+    image: jaegertracing/all-in-one:latest
```

Comment:
Do we want to specify `docker.io` registry for the image here as in other cases in `docker-compose.yml` so during the run we are not prompt  to pick one?
```
? Please select an image: 
    registry.fedoraproject.org/jaegertracing/all-in-one:latest
    registry.access.redhat.com/jaegertracing/all-in-one:latest
  ▸ docker.io/jaegertracing/all-in-one:latest
    quay.io/jaegertracing/all-in-one:latest
```

Reply:
Good idea! Updated.